src/components/ToDoAlert/ToDoAlert.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+        <Transition @after-leave="removeAlert">
+            <div v-if="showAlert" :class="setWrapperClass">
+                <div class="alertWrapper__alertImgContainer">
+                    <img class="alertWrapper__alertImg" :src="setSvgImage">
+                </div>
+                <div :class="setMessageContainerClass">
+                    <h3 class="alertWrapper__header">{{message.title}}</h3>
+                    <p>{{ message.text }}</p>
+                </div>
+                <div class="alertWrapper__btnContainer">
+                    <ToDoButton
+                        :btnClass="setBtnClass"
+                        @click="handleDeleteClick"
+                    />
+                </div>
+            </div>
+        </Transition>
+</template>
+<script>
+import '@/components/ToDoAlert/ToDoAlert.scss';
+import infoImg from '@/components/assets/infoImg.svg';
+import errorImg from '@/components/assets/errorImg.svg';
+import ToDoButton from '@/components/ToDoButton/ToDoButton.vue';
+
+export default {
+    data() {
+        return {
+            showAlert: false,
+            infoImg: infoImg,
+            errorImg: errorImg,
+            timerId: null,
+        };
+    },
+    props: {
+        message: Object,
+        type: String,
+        persistent: Boolean,
+    },
+    components: {
+        ToDoButton,
+    },
+    mounted() {
+        this.showAlert = true;
+
+        if (!this.persistent) {
+            this.timerId = setTimeout(() => {
+                this.showAlert = false;
+            }, 3000);
+        }
+    },
+    beforeUnmount() {
+        clearTimeout(this.timerId);
+    },
+    computed: {
+        setWrapperClass() {
+            if (this.showAlert) {
+                if (this.type === 'error') {
+                    return ['alertWrapper', 'alertWrapper--error'];
+                } else if (this.type === 'info') {
+                    return ['alertWrapper', 'alertWrapper--info'];
+                }
+            }
+            return null;
+        },
+
+        setMessageContainerClass() {
+            if (this.showAlert) {
+                if (this.type === 'error') {
+                    return ['alertWrapper__messageContainer', 'alertWrapper__messageContainer--error'];
+                } else if (this.type === 'info') {
+                    return ['alertWrapper__messageContainer', 'alertWrapper__messageContainer--info'];
+                }
+            }
+            return null;
+
+        },
+
+        setSvgImage() {
+            if (this.type === 'error') {
```

Comment:
`if (this.type === 'error') return this.errorImg;
 if (this.type === 'info')  return this.infoImg;
 return null;`

Reply:
zajebiscie to sformatowalem!

Reply:
chyba jeszcze lepiej to sformatowałem nie?